WebApp/server/image-processing.coffee

```diff
@@ -14,7 +14,7 @@ _app.createThumbnails = (collection, fileRef, cb) ->
     return true
```

Comment:
Consider using a more descriptive error message. For example, 'File not found error: The file #{fileRef.path} was not found in the [createThumbnails] method.'